docs/docs/getting-started/integration.md

```diff
@@ -25,9 +25,56 @@ var myChart = new Chart(ctx, {...});
 Chart.js 3 is tree-shakeable, so it is necessary to import and register the controllers, elements, scales and plugins you are going to use.
 
```

Comment:
Maby add a line under here with something along the line with: For all available imports see the example below.